tests/conftest.py

```diff
@@ -34,14 +45,20 @@
 
 
 class LocalFileAdapter(requests.adapters.BaseAdapter):
-    def __init__(self, base_path):
+    def __init__(self, base_path: PosixPath) -> None:
```

Comment:
```suggestion
    def __init__(self, base_path: Path) -> None:
```

Reply:
I wondered if Path is correct since LocalFileAdapter is used only by `get_local_finder`, but as thinking of scalability I'm going to fix this.

Reply:
fixed here

thank you for using suggested change, but it may be differ from flake8 and I wanted if mypy pass through, I fixed on local.

[`271559d` (#354)](https://github.com/pdm-project/pdm/pull/354/commits/271559d31a7363e60d2c95e51972ad9654e18fbb#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128R58)